presentation/components/src/test/common/UnifiedSelectionContext.test.tsx

```diff
@@ -103,7 +106,7 @@ describe("UnifiedSelectionContext", () => {
     expect(firstResult.removeFromSelection).to.be.equal(secondResult.removeFromSelection);
   });
 
-  it("does not update context when selection changes on one level below", () => {
+  it("does not update context when selection changes one level deeper", () => {

```

Comment:
I think this is correct, but this behavior should be mentioned explicitly next to `UnifiedSelectionContextProviderProps.selectionLevel`. The docs should say that the context will change if selection changes at or lower than specified selection level, but not higher.

Reply:
I thought this was implied, because the provider watches a specific selection level. Updated documentation nevertheless in b60e883094f695f809a74edd484f043e532ec4a2.